tdrs-frontend/package.json

```diff
@@ -6,13 +6,19 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.16",
+    "ansi-regex": "^5.0.1",
     "axios": "^0.24.0",
+    "browserslist": "^4.16.5",
     "classnames": "^2.3.0",
     "connected-react-router": "^6.8.0",
     "file-type": "^16.5.3",
+    "glob-parent": "^5.1.2",
     "history": "^4.7.2",
+    "immer": "^9.0.6",
     "include-media": "^1.4.10",
+    "json-schema": "^0.4.0",
     "miragejs": "^0.1.42",
+    "nth-check": "^2.0.1",
```

Comment:
These should not be added as new direct dependencies (because they aren't); this PR should only include the lockfile changes.